src/services/api-client.js

```diff
@@ -0,0 +1,30 @@
+const BASE_PATH = 'http://localhost:3000'
+
+class ApiClient {
+  login(username, password) {
+    const params = `username=${username.value}&password=${btoa(password.value)}`
+    return fetch(`${BASE_PATH}/users?${params}`).
+      then(res => {
+        if(res.ok) return res.json()
+        throw res.err
+      })
+  }
+
+  fetchSites(user) {
+    return fetch(`http://localhost:3000/sites?owner=${user}`).
```

Comment:
I believe this should be:
```suggestion
    return fetch(`${BASE_PATH}/sites?owner=${user}`).
```